src/custom-widget.ts

```diff
@@ -79,40 +72,44 @@ window.addEventListener("message", receiveMessage, false);
  * Send message to parent component
  * @param message message to send
  */
-const sendMessage = (message: IMessage): void => {
+const sendMessage = (message: TMessage): void => {
   window.parent.postMessage(message, "*");
 };
 
-window.TagoIO.ready = (options) => {
```

Comment:
The assignments/bindings to the `window.TagoIO` object were moved to the end of the file so that everything is in one place.